WebKitBrowser/WebKitImplementation.cpp

```diff
@@ -573,6 +577,8 @@ static GSourceFuncs _handlerIntervention =
             Core::JSON::Boolean ThunderDecryptorPreference;
             Core::JSON::String MemoryProfile;
             Core::JSON::String MemoryPressure;
+            Core::JSON::DecUInt32 WebProcessLimit;
```

Comment:
Let's also use  #ifdef WEBKIT_MEMORY_PRESSURE_API here. Not all three members are needed at the same time. Either the old is supported or the new one, but not both at the same time. This makes the code more efficient and also makes that concept more clear

Reply:
done.